aspnetcore/getting-started.md

```diff
@@ -18,23 +18,25 @@ uid: getting-started
 
 2.  Create a new .NET Core project:
 
-    ```console
+    ```terminal
```

Comment:
"terminal" isn't in the list of supported language types at http://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases but then console isn't either.